### ISSUE TITLE
dev-java/hsdis: eselect hsdis list won't find icedtea java

### ISSUE THREAD

**Issue by Niko-sk2x** [no-hope/fw-overlay#4]
Based on my very limited understanding of this part:
```bash
find_targets() {
        local bcdir
        for bcdir in ${EROOT%/}${_jvm_dir}/*
        do
                [[ -e ${bcdir} && "$(${bcdir}/bin/java -version 2>&1)" =~ .*HotSpot.* ]] && basename "${bcdir}"
        done | sort -u
}
```
I think it attempts to find `HotSpot` in `java -version` output, but for (at least my installation) icedtea it's not there.

My `java -version` output:

```
openjdk version "1.8.0_111"
OpenJDK Runtime Environment (IcedTea 3.2.0) (Gentoo icedtea-3.2.0)
OpenJDK 64-Bit Server VM (build 25.111-b14, mixed mode)
```

**Comment by pshirshov** [comment]
Probably it's good idea to use Oracle JDK to run IDEA.

**Comment by KetothXupack** [comment]
Unfortunately (afaik) hsdis refused to run on top of IcedTea. Let me re-check that

**Comment by KetothXupack** [comment]
@Barteks2x Turns out that it works! 

For some reason IcedTea fails to locate library under `/usr/lib/jvm/icedtea-bin-XXX/lib/amd64/` with `Could not load hsdis-amd64.so; library not loadable; PrintAssembly is disabled` error message. So I have to move it under `/usr/lib/jvm/icedtea-bin-XXX/jre/lib/amd64/` and now both Oracle HotSpot and IcedTea happy.

Fixed in `>=dev-java/hsdis-8.74.02-r1`
